web/src/components/GuidanceListPage/GuidanceListPage.tsx

```diff
@@ -89,16 +89,17 @@ export const getGuidanceListPage =
 		// Announcement text, used for giving audible notifications to screen readers when results have changed
 		const [announcement, setAnnouncement] = useState(""),
 			// Cache the breadcrumbs as they're static and it means we can use them on both the error view and success view
-			breadcrumbs = useMemo(
-				() => (
-					<Breadcrumbs>
-						<Breadcrumb to="/">Home</Breadcrumb>
-						<Breadcrumb to="/guidance">NICE guidance</Breadcrumb>
-						<Breadcrumb>{breadcrumb}</Breadcrumb>
-					</Breadcrumbs>
+			MemoizedBreadcrumbs = () =>
```

Comment:
I turned it into a custom component (assigned to a fat arrow) so we could use it like <MemoizedBreadcrumbs  /> in the template. The React jsx requires the capitalisation. 

I can remove the fat arrow and keep the name but it will be {memoizedBreadcrumbs} versus <MemoizedBreadcrumbs /> in the template render?